README.md

```diff
@@ -20,18 +20,40 @@ js-libp2p-bootstrap
 ## Usage
 
 ```JavaScript
+const Libp2p = require('libp2p')
 const bootstrap = require('libp2p-bootstrap')
-
-const options = {
-  list: <List of Multiaddrs>
-  interval: 5000 // ms, default is 10s
+const TCP = require('libp2p-tcp')
+const { NOISE } = require('libp2p-noise')
```

Comment:
```suggestion
const { NOISE } = require('libp2p-noise')
const { MPLEX } = require('libp2p-mplex')

```

Reply:
A stream multiplexer is not technically required, but is highly recommended. So, let's also add it